examples/go/go.mod

```diff
@@ -3,14 +3,13 @@ module twingate_go_example
 go 1.22.0
 
 require (
-	github.com/Twingate/pulumi-twingate/sdk/v3 v3.0.4
-	github.com/pulumi/pulumi/sdk/v3 v3.116.1
+	github.com/Twingate/pulumi-twingate/sdk/v3 v3.0.0-20240707084841-6cd0e7a1501f
```

Comment:
shouldn't this be 3.0.6?

Reply:
It should but that version did not show up in https://pkg.go.dev/github.com/Twingate/pulumi-twingate/sdk/v3 . I actually had to request that path to be scanned by pkg.go.dev

Hopefully maybe the next release it will publish a matching version